script/Script.py

```diff
@@ -0,0 +1,13 @@
+import MonthCalculation
+import datetime
+import GraphQLQuery
+import Log
+import PSQL
+
+def main():
+    Log.clearLog()
+    today = datetime.datetime.now()
+    for i in range(0, 5):
```

Comment:
This should probably be configurable, but a default of 120 (10 years) would be a good starting point.